### ISSUE TITLE
[Request]: Ability to mute a conversation

### ISSUE THREAD

**Issue by algernon** [GeopJr/Tuba#1290]
### Describe the request

Both Mastodon and GoToSocial allow me to mute a conversation I am participating in, which is *very* handy when I wish to stop receiving notifications about them. It would be nice if Tuba would allow me to do the same.

On both GtS and Mastodon, the api is [`/api/v1/statuses/{id}/mute`](https://docs.joinmastodon.org/methods/statuses/#mute) (& `/unmute`), and is available for any status that is owned by me, or mentions me.

### Implementation Details

- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by GeopJr** [comment]
I did add it on 0.9 but only now realized (? unless I'm wrong?) that you can mute any convo you are mentioned in too. Currently, you can mute conversations if you are the author of the post (there should be a mute/unmute button in the post menu)

The docs claim "Do not receive notifications for the thread that this status is part of. Must be a thread in which you are a participant." **but** what I followed was Mastodon web, which does it the same way (only shows "Mute Conversation" if you are the author).

I guess I'll test it and see, if I can mute it just by being mentioned I will push it in the patch release!

**Comment by algernon** [comment]
> Currently, you can mute conversations if you are the author of the post (there should be a mute/unmute button in the post menu)

Oh! I can, indeed, mute conversations starting from a toot I authored.

> what I followed was Mastodon web, which does it the same way (only shows "Mute Conversation" if you are the author).

I see the "mute/unmute" option on Mastodon's web UI for statuses where I'm merely mentioned in, but am not the author of:

![image](https://github.com/user-attachments/assets/e85d8c23-1f2c-4167-9e65-61b4fea8469d)

(This is on my old glitch-soc instance, so it *might* be a glitch-soc thing.)

**Comment by GeopJr** [comment]
It doesn't for me :/


https://github.com/user-attachments/assets/f2323ba7-8de7-45bc-bdb7-a66038f794d3


https://github.com/user-attachments/assets/a94c73d9-a48e-41ee-9862-eff5320be88c

I can mute the convo from my post but not from the one mentioning me?!? :shrug: as long as the API works, I'll enable it for mentions too

**Comment by algernon** [comment]
Huh, weird. I just tried it on a stock mastodon server (fosstodon), and I see it there, too:

![image](https://github.com/user-attachments/assets/fffab235-e99e-480c-a3df-3523999936c2)

Nevertheless: when I tried muting a conversation via another client (mastodon's frontend, hosted at https://masto-fe.iceshrimp.dev), muting a conversation via a status I was only mentioned in did work on my GtS instance, I stopped receiving notifications about further messages. So it works with GtS at least! :)
